Blackjack/Controller.cs

```diff
@@ -73,24 +73,25 @@ public void Play()
                 if (choice == Choice.Hit) 
                 {
                     var playerCard = DealCard(player);
-                    _output.WriteLine(String.Format(Messages.PlayerCard, OutputFormatter.DisplayCard(playerCard))); // TODO: in test card is null
+                    _output.WriteLine(String.Format(Messages.PlayerCard, OutputFormatter.DisplayCard(playerCard))); 
                 }
                 // calculate player's score
                 playerScore = Score.Calculate(player.Hand);
                 _output.WriteLine(String.Format(Messages.PlayerScore, playerScore, OutputFormatter.DisplayHand(player.Hand)));
             }
-            while (choice != Choice.Stay);
+            while (!EndTurn(choice, playerScore));

```

Comment:
name should reflect that it returns a boolean i.e. `EndTurn` sounds like it will do something rather than check something